custom_components/plugwise/climate.py

```diff
@@ -154,11 +161,19 @@ def hvac_action(self) -> HVACAction:
         ]
         if hc_data["binary_sensors"]["heating_state"]:
             return HVACAction.HEATING
-        if hc_data["binary_sensors"].get("cooling_state", False):
+        if hc_data["binary_sensors"].get("cooling_state", False):  # pw-beta adds False
             return HVACAction.COOLING
 
         return HVACAction.IDLE
 
+    @property
+    def extra_state_attributes(self) -> Mapping[str, Any] | None:
+        """Return entity specific state attributes."""
+        return {
+            "available_schemas": self.device["available_schedules"],
+            "selected_schema": self.device["selected_schedule"],
+        }
+
```

Comment:
Don't add this back please. To be deleted in Core.
We have a `select` replacing this.